bin/sofa_utils.py

```diff
@@ -0,0 +1,27 @@
+import json
+import pandas as pd 
+import itertools
+import os
+from collections import defaultdict, OrderedDict
```

Comment:
Please group and sort the imports. (see also [flake8-import-order](https://github.com/PyCQA/flake8-import-order))

Grouping them as:
1. stdlib first
2. 3rd party library second
3. pkg/mod from sofa itself last

Sorting:
Either:
a) `import` first, then `from ... import ...` second
b)  sort by package name lexically, no matter `import` or `from ...`